target/scripts/helpers/relay.sh

```diff
@@ -0,0 +1,136 @@
+#! /bin/bash
+# Support for Relay Hosts
+
+function _relayhost_default_port_fallback
+{
+  [[ -z ${RELAY_PORT} ]] && RELAY_PORT=25
```

Comment:
That's, how we set defaults in other places:

```suggestion
  : ${RELAY_PORT:=25}
```

```suggestion
  RELAY_PORT=${RELAY_PORT:-25}
```

Reply:
This was the original line already in use, I started making some minor changes with relay logic which was the first to extract, but then realized it was better at least for review to minimize introducing changes :sweat_smile: 

I can apply this though no worries :+1: